tools/ci/check_grep.sh

```diff
@@ -218,7 +218,7 @@ while read f; do
         echo -e "${RED}ERROR: file $f is missing a trailing newline${NC}"
         st=1
     fi;
-done < <(find . -type f -not \( -path "./.git/*" -prune \) -exec grep -Iq . {} \; -print)
+done < <(find . -type f -not \( -path "./.git/*" -prune \) -not \( -path "./tgui/.yarn/*" -prune \) -exec grep -Iq . {} \; -print)
```

Comment:
what's this about?

Reply:
Previous runs didn't have to account for that folder because the yarn fetch happened afterwards. Now it happens beforehand and not all of the fetched files are going to have newlines